### ISSUE TITLE
⌨ Accessibility-related copy issue

### ISSUE THREAD

**Issue by RM-1978** [GCTC-NTGC/gc-digital-talent#5379]
## ❌ Current behaviour (issue)
Users are not sure why the introductory text on this page doesn't tell them anything about the content and interactions on the page.

### How to reproduce
**AT Configuration** JAWS, NDA, Narrator, VoiceOver in Windows 10/11 or MacOS
1. Open JAWS, NVDA or narrator 
2. Navigate to the Browse Jobs page. 
3. Listen to or read the copy to understand what is the  nature of the page and what you are supposed to do on it. 

## ✔ Expected behaviour (fix)
Browse Jobs copy should be updated to reflect the content and the interactions on the page so that the user knows what to expect on this page. 

**Suggested copy addition**
This platform allows you to apply to recruitment processes that make it easy for hiring managers to find you. _**This page offers you a snapshot of our open recruitment processes. Come back and check this page often!**_

Your application to a process will be reviewed by our team and if it's a match, you will be invited to an assessment. Once accepted, managers **can** contact you about job opportunities. 
 
### Acceptance Criteria
A set of assumptions which, when tested, verify that the accessibility requirement was met.
- [ ] it is clear to all users (including those using ATs) what this page is and how they can interact with it.

**Comment by substrae** [comment]
I'm fine with this as well, though I would suggest some minor edits:

> Applying to our recruitment processes make it easy for hiring managers to find you. This page provides you with a summary of our open recruitment processes. Come back and check this page often!
>
> Your application to a recruitment will be reviewed by our team and, if it's a match, you will be invited to an assessment. Once you're accepted, you'll appear in a list of talent that managers can contact about job opportunities.

**Comment by petertgiles** [comment]
Thanks, I've updated the issue and tried to merge Rosalie's formatting and Josh's copy editing.  Feel free to update if I've not got it quite right.

**Comment by mnigh** [comment]
**FR** copy returned from translation services:

```
"C7sYnb": {
 "defaultMessage": "Cette plateforme vous permet de postuler à des processus de recrutement qui permettront aux gestionnaires qui embauchent de vous trouver facilement. Cette page donne un aperçu de nos processus de recrutement ouverts. Revenez-y et consultez cette page fréquemment!",
 "description": "Description of how the application process works, paragraph one"
},
"vNFHWp": {
 "defaultMessage": "Votre demande à un processus sera examinée par notre équipe et, si nous trouvons une concordance quelconque, on vous invitera à une évaluation. Une fois accepté(e), les gestionnaires pourront alors communiquer avec vous avec des offres d’emploi.",
 "description": "Description of how the application process works, paragraph two"
},
```
